src/pages/Lend/Filter/FilterSections/LocationMulti/LocationMultiRefinements.vue

```diff
@@ -30,6 +31,7 @@
 					<div
 						v-if="!item.allCountriesSelected"
 						@click="selectAllCountries(item.countries, item.region)"
+						id="select-all"
```

Comment:
if possible please add the region to this (select-all-##region##) and to the deselection toggle as well.

Reply:
Done